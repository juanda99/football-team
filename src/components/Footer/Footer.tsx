import React from 'react'
import Typography from '@mui/material/Typography'
import Link from '@mui/material/Link'
import Box from '@mui/material/Box'

const Copyright = (): JSX.Element => {
  return (
    <Typography variant="body2" color="text.secondary" align="center">
      {'Copyright © '}
      <Link color="inherit" href="https://www.adidas.es/">
        Adidas
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  )
}

const Footer = (): JSX.Element => {
  return (
    <Box sx={{ bgcolor: 'background.paper', p: 6 }} component="footer">
      <Typography variant="h6" align="center" gutterBottom>
        Adidas Challenge
      </Typography>
      <Typography
        variant="subtitle1"
        align="center"
        color="text.secondary"
        component="p"
      >
        Juan Daniel Burró Aláez - May, 2022
      </Typography>
      <Copyright />
    </Box>
  )
}
export default Footer
