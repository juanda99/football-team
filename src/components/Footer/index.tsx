import React from 'react'
import Typography from '@mui/material/Typography'
import Link from '@mui/material/Link'
import Box from '@mui/material/Box'

const Copyright: React.FC = () => {
  return (
    <Typography variant="body2" color="text.secondary" align="center">
      {'Copyright © '}
      <Link color="inherit" href="https://www.adidas.es/">
        Adidas
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  )
}

export const Footer: React.FC = () => {
  return (
    <Box sx={{ bgcolor: 'background.paper', p: 6 }} component="footer">
      <Typography variant="h6" align="center" gutterBottom>
        Challenge ADIDAS
      </Typography>
      <Typography
        variant="subtitle1"
        align="center"
        color="text.secondary"
        component="p"
      >
        Juan Daniel Burró Aláez
      </Typography>
      <Copyright />
    </Box>
  )
}
