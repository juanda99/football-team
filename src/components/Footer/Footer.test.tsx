import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'

describe('Footer component', () => {
  it('renders correctly', () => {
    const elem = renderer.create(<Footer />).toJSON()
    expect(elem).toMatchSnapshot()
  })
})
