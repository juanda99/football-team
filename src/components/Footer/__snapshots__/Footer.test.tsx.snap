// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<footer
  className="MuiBox-root css-16ybtzv"
>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-alignCenter MuiTypography-gutterBottom css-tcrtuv-MuiTypography-root"
  >
    Adidas Challenge
  </h6>
  <p
    className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter css-1xatcrf-MuiTypography-root"
  >
    Juan Daniel Burró Aláez - May, 2022
  </p>
  <p
    className="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
  >
    Copyright © 
    <a
      className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1cghntr-MuiTypography-root-MuiLink-root"
      href="https://www.adidas.es/"
      onBlur={[Function]}
      onFocus={[Function]}
    >
      Adidas
    </a>
     
    2022
    .
  </p>
</footer>
`;
